### PR TITLE
TC-OO-2.3: Wait 10 seconds before performing step 21b

### PR DESCRIPTION
**Pull request by bauerschwan**
see https://github.com/CHIP-Specifications/chip-test-plans/issues/2024 and https://github.com/CHIP-Specifications/chip-test-plans/pull/2025
 
In the OnOff test plan a wait time of 10 seconds was forgotten before step 21b.
This PR adds the missing 10 seconds wait time before step 21b
within the timer callback

### CODE REVIEW

**Review by CKnoerzer — APPROVED** [review]

File: src/app/tests/suites/certification/Test_TC_OO_2_3.yaml [hunk 1]
```diff
@@ -866,6 +866,14 @@ tests:
               - name: "OffWaitTime"
                 value: 300
 
+    - label: "Wait 10000ms"
+      cluster: "DelayCommands"
+      command: "WaitForMs"
+      arguments:
+          values:
+              - name: "ms"
+                value: 10000
```
Comment (tcarmelveilleux) [inline src/app/tests/suites/certification/Test_TC_OO_2_3.yaml:875]: This PR is missing the code regeneration. If no other files than the YAML are modified, it means the tests are not actually modified. Please run `scripts/tools/zap_regen_all.py --type tests`.

Comment (bauerschwan) [reply src/app/tests/suites/certification/Test_TC_OO_2_3.yaml:875]: Thx!
will do so...

Comment (bauerschwan) [reply src/app/tests/suites/certification/Test_TC_OO_2_3.yaml:875]: Done.

**Review by bzbarsky-apple — COMMENTED** [review]
Needs to rebase to tip and regen chip-tool and darwin-framework-tool....

**Comment by bauerschwan** [comment]
> Needs to rebase to tip and regen chip-tool and darwin-framework-tool....

Thanks, @bzbarsky-apple! 
Done that. Now only one test fails but looks like this is unrelated to my change.
What next?

**Comment by woody-apple** [comment]
> > Needs to rebase to tip and regen chip-tool and darwin-framework-tool....
> 
> Thanks, @bzbarsky-apple! Done that. Now only one test fails but looks like this is unrelated to my change. What next?

I re-kicked CI, and it should be good to go.

**Pull request merged**